### ISSUE TITLE
Sprite SVG injected into the DOM needs aria-hidden=true for Accessibility 

### ISSUE THREAD

**Issue by jarodtaylor** [JetBrains/svg-sprite-loader#315]
**Do you want to request a feature, report a bug or ask a question?**

I'm having [issues running the bootstrap](https://github.com/kisenka/svg-sprite-loader/issues/303) to create my own PR, and it would probably be quicker for you to just add a hotfix  (I believe it's actually part of svg-baker-runtime) that adds the `aria-hidden=true` to the SVG markup that is injected into the DOM. This will aid in Accessibility and tell screen readers that this content is hidden.

**Comment by basham** [comment]
My team was likewise encountering this issue. We noticed that when analyzing our site with the AXE plugin for Chrome, it warned that the SVG sprite was a child of `<body>` instead of some region element ([*All page content must be contained by landmarks*](https://dequeuniversity.com/rules/axe/3.3/region?application=AxeChrome)). Not having content in landmark regions makes it difficult for screen reader users to find the content. However, SVG sprites are not useful for screen readers because it's there to support the actual content. There's no value in a screen reader user or any other user to know about this SVG sprite. We recommend that these SVG sprites add the attribute `aria-hidden="true"` to fix this issue.

As a temporary workaround, we could potentially reconfigure the loader to mount the sprite in a custom element that is hidden from screen readers, using the [`spriteModule` configuration](https://github.com/kisenka/svg-sprite-loader#spritemodule-autoconfigured).

```html
<body>
  <div id="my-custom-mounting-target" aria-hidden="true"></div>
<body>
```

```js
domready(() => sprite.mount('#my-custom-mounting-target'))
```

Another temporary solution would be to add some script that would dynamically add `aria-hidden="true"` to the `<svg>` once the sprite has been loaded to the DOM.

**Comment by pwavg** [comment]
Any progression on this?

**Comment by blissdev** [comment]
Running into this issue as well with axe and our accessibility testing.

**Comment by blissdev** [comment]
Another, albeit brittle solution, is to manually place an SVG element in your body with `id="__SVG_SPRITE_NODE__"` with the same attributes as the one it places currently but with an additional `aria-hidden` and it will re-use that element instead of creating a new one.

**Comment by kisenka** [comment]
Fixed in svg-sprite-loader@4.2.2
